apps/cowswap-frontend/src/modules/limitOrders/utils/limitOrderSettingsAnalytics.ts

```diff
@@ -0,0 +1,56 @@
+import { useCowAnalytics } from '@cowprotocol/analytics'
+
+import { CowSwapCategory } from 'common/analytics/types'
+
+export enum LimitOrderSettingsAction {
+  TOGGLE_SETTINGS = 'Toggle Limit Order Settings',
+  CUSTOM_RECIPIENT = 'Custom Recipient',
+  PARTIAL_EXECUTIONS = 'Enable Partial Executions',
+  PRICE_POSITION = 'Limit Price Position',
+  LOCK_PRICE = 'Lock Limit Price',
+  USD_MODE = 'Global USD Mode',
+  TABLE_POSITION = 'Orders Table Position',
+}
+
+export function useLimitOrderSettingsAnalytics() {
```

Comment:
This is a hook, the file name should have the same name and the file should be in `hooks` directory instead of `utils`

Reply:
Addressed